index.html

```diff
@@ -2382,8 +2400,7 @@ <h2>Secure Practices for end-to-end security</h2>
 	      <a href="https://datatracker.ietf.org/doc/rfc7515/">RFC 7515 (IETF JSON Web Signature)</a>
 	      provides guidelines for computation
               and validation of digital signatures or MACs using JSON-based data structures. 
-            </p>
-            <p>
+           

```

Comment:
probably should structure all these examples into a bulleted list rather than merging them into one big and hard to read paragraph.